internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -50,15 +50,20 @@ var Tables = []any{
 }
 
 type Build struct {
-	BuildID   sdktypes.UUID `gorm:"primaryKey;type:uuid;not null"`
+	BuildID   sdktypes.UUID  `gorm:"primaryKey;type:uuid;not null"`
+	ProjectID *sdktypes.UUID `gorm:"index;type:uuid"`
```

Comment:
I think we can index not null, it will make the index smaller
could you check if it is viable for this case?

Reply:
ENG-1093